data/mt/retired/Dennis-Lenz-0fd1ad64-349c-42d9-ae46-4ff7d3eb9495.yml

```diff
@@ -11,7 +11,8 @@ roles:
   type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '50'
-- type: lower
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person is in the pull twice. They will be in the house until 01/02/2023 and then start in the senate on that date.